spec/section_spec.rb

```diff
@@ -47,6 +47,21 @@ class PageWithSection < SitePrism::Page
       end
     end
 
+    context 'name argument does not exist' do
+      it 'should generate class name from section name' do
+        class HeaderSection < SitePrism::Section
+        end
+
+        class Page < SitePrism::Page
+          section :header, '.header'
+        end
+
+        page = Page.new
+
+        expect(page).to respond_to :has_header?
+      end
+    end
+
```

Comment:
I'd like to see a test showing that a named argument does work.  Also, I'm not sure that using just `respond_to?` is sufficient.  That checks that the method was corrected, but never actually calls the method's code.  Given the state of the test suite, (and the metaprogrammy nature of site_prism) there may be little choice but to test that in the cucumber tests.  :-/